crates/rpc/src/lib.rs

```diff
@@ -1,14 +1,25 @@
-pub fn add(left: u64, right: u64) -> u64 {
-    left + right
-}
+use std::{net::SocketAddr, sync::Arc};
+
+use config::ServerConfig;
+use routes::get_routes;
+use types::genesis::Genesis;
+use utils::error::handle_rejection;
+use warp::Filter;
+
+pub mod config;
+pub mod handlers;
+pub mod routes;
+pub mod types;
+pub mod utils;
+
+/// Start the Beacon API server.
+pub async fn start_server(ctx: Arc<Genesis>, server_config: ServerConfig) {
+    let addr: SocketAddr = format!("{}:{}", server_config.http_address, server_config.http_port)
+        .parse()
+        .expect("Unable to read ServerConfig");
 
-#[cfg(test)]
-mod tests {
-    use super::*;
+    let routes = get_routes(ctx).recover(handle_rejection);
 
-    #[test]
-    fn it_works() {
-        let result = add(2, 2);
-        assert_eq!(result, 4);
-    }
+    println!("Starting server on {}", addr);
+    warp::serve(routes).run(addr).await;
```

Comment:
```suggestion
    info!("Starting server on {}", server_config.http_socket_address);
    serve(routes).run(server_config.http_socket_address).await;
```

Please don't use print for logs, use the `tracing` crate